deploy/prod-fss-default.yml

```diff
@@ -65,6 +65,8 @@ spec:
       value: "https://psb-inntektsmelding-frontend.intern.nav.no/"
     - name: PSB_ETABLERT_TILSYN_FRONTEND_URL
       value: "https://psb-etablert-tilsyn-frontend.intern.nav.no/"
+    - name: PSB_OM_BARNET_FRONTEND_URL
+      value: "http://k9-sak"
```

Comment:
I prod må denne peke på hva enn ingressen på frontendappen er i prod :)
Se https://github.com/navikt/psb-om-barnet-frontend/blob/main/nais/prod-fss.json#L8

Reply:
tenkte å vente med denne til vi er klar for prod, men kanskje ingen grunn til det

Reply:
Hvis ikke det er lagt ut i P ennå kan vi sikkert vente :)